chaossearch/resource_object_group.go

```diff
@@ -65,6 +65,13 @@ func resourceObjectGroup() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validation.StringIsValidRegExp,
 			},
+			"pattern": {
+				Type:         schema.TypeString,
+				Default:      ".*",
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringInSlice([]string{"json", "log"}, true),
```

Comment:
this makes sure the pattern is either `log` or `json`, right? Is that what we want?

Reply:
Yes, it checks that the string is in array